assignment/index.html

```diff
@@ -62,21 +62,59 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
+      var jsonToCsv = function(json){console.log(json);};
+
+      let healthCentersInfo = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']];
+      for (var i=0; i<healthCenters.length; i++) {
+          var health = [healthCenters[i].LNG,
+          healthCenters[i].LAT,
+          healthCenters[i].OBJECTID,
+          healthCenters[i].NAME,
+          healthCenters[i].ORGANIZATION,
+          healthCenters[i].ZIP,
+          healthCenters[i].PHONE,
+          healthCenters[i].WEBSITE,
+          healthCenters[i].DENTAL_PHONE,
+          healthCenters[i].FULL_ADDRESS];
+          healthCentersInfo.push(health)
+      };
+
+      var icon1 = L.icon({
+        iconUrl:'D:/2019Spring/CPLN692/Data/cpln692-week2/assignment/images/marker-icon.png',
```

Comment:
When I try to run this on my computer. It does not work because this pathway is absolute and not relative.

If you change it to... `iconUrl:'images/marker-icon.png',` 

it works!